src/product/dashboard.njk

```diff
@@ -76,13 +76,14 @@ meta:
                 <p class="mt-6">
                     Get started quickly with your new FlowFuse Dashboard. This migration service will automatically convert
                     your original Node-RED Dashboard flows, and make them FlowFuse Dashboard-ready.
-                </p>
-                <p class="pb-5">
-                    This service will improve over time. Currently, we support the automatic conversion of
-                    <code>ui_tab</code>, <code>ui_link</code> and <code>ui_group</code> nodes.
-                </p>
-            </div>
+                </p></div>
```

Comment:
Not sure why?

Reply:
yep, good spot, reverting. Not sure how that happened